projects/pokt.network/package.yml

```diff
@@ -0,0 +1,28 @@
+distributable:
+  url: https://github.com/pokt-network/pocket-core/archive/refs/tags/RC-{{version}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: https://github.com/pokt-network/pocket-core/tags
```

Comment:
Only the org/repo without GitHub.com and https

Reply:
Works!